### PR TITLE
Capture website streams for analytics

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
* to capture activities in WP sites
  - browsing product (no more than once per 1 minute)
  - password reset REQUEST (ditto)
  - password reset SUCCESS (ditto)

### What:

- added cron job to submit captured activity streams

alt text

alt text

### Refs:

- https://app.asana.com/0/939825968139797/1126361313453039
- https://troly.aha.io/features/APP-278

### CODE REVIEW

**Comment by JamesEmpireOne** [comment]
We will need to add a new `Stream.what` type @atifempireone  - `analytics` then break it down into the operations `viewed`/`add_to_cart`/`remove_from_cart` etc...

File: includes/frontend/controllers/class-wp99234-wc-filter.php [hunk 1]
```diff
@@ -1201,6 +1225,175 @@ function handle_ratings_and_reviews( $id, $comment ){
...
+                    echo "Partial form sign-up";
+                } else {
+                    echo "Already started signing-up in Club membership";
+                }
+            }
+        } else {
+            echo "User is not login yet";
+        }
+
+        wp_die();
```
Comment (JamesEmpireOne) [inline includes/frontend/controllers/class-wp99234-wc-filter.php]: Is there a reason this is here @rcdtroly ?

**Comment by JamesEmpireOne** [comment]
@rcdtroly You'll need to just fix up some final changes before testing against staging.

`what` must always be `analytics`
`src` must always be `website`
`operation` is what is used to show the `event` itself

For `operation`, can you please ensure that the generic stuff like `add_to_cart` and not `woocommerce_add_to_cart` that is being processed.

**Comment by rcdtroly** [comment]
@JamesEmpireOne fixed - assign user to streams upon `Placing Order` and optimised code

**Pull request merged**